### DISCUSSION TITLE
Testnet tokens don't arrive

### DISCUSSION THREAD

**Discussion by FaradayNodes** [entropyxyz/community#124] [Support]
We have [requested testnet tokens](https://github.com/entropyxyz/community/discussions/122) but haven't hit our wallet.
This error shows when we try to see the tx

Unable to retrieve the specified block details. 4003: Client error: Api called for an unknown Block: State already discarded for 0xdda110db98d36e1d8014f37a41af97a14104756838185b59ca14565d0e1cbae2

If you could please send some we appreciate. 
Faraday

**Comment by johnnymatthews** [comment]
The testnet was recently reset, so any tokens requested _before_ last weekend will have been wiped out, and accounts reset. Read the announcement for more details: https://github.com/entropyxyz/community/discussions/123

**Reply by FaradayNodes** [reply]
Hi,

Testnet tokens were requested yesterday

**Reply by johnnymatthews** [reply]
Looked into this a bit further.

> Unable to retrieve the specified block details. 4003: Client error: Api called for an unknown Block: State already discarded for 0xdda110db98d36e1d8014f37a41af97a14104756838185b59ca14565d0e1cbae2

It looks like the blockchain explorer isn't able to retrieve data from more than a couple hours ago. I'll pass this onto the devs and get this addressed.

In the meantime, can you double-check that the address you gave in your original request is, in fact, the correct address. If it is, and you still can't see any funds in your account, create a new address and send that over to me.

**Reply by FaradayNodes** [reply]
Hi, 

Double checked and no tokens hit the talisman wallet with faucet enabled.
Can you please try and send some to the same address 5EqUREuFWVDD16CUKvverPmgBAezFMexHKFbjVbponSq9ioi ?

Thank you

**Reply by johnnymatthews** [reply]
Just sent some more. They should be in your account now.

**Comment by johnnymatthews** [comment]
It's been a while since there were any posts in this discussion. Closing.
